packages/@functionless/aws-dynamodb-constructs/src/runtime.ts

```diff
@@ -1,64 +1,28 @@
-import { aws_dynamodb } from "aws-cdk-lib";
-import { JsonFormat } from "typesafe-dynamodb";
-import {
+import type {
   BatchGetItem,
-  BatchGetItemInput,
-  BatchGetItemOutput,
-  createBatchGetItemIntegration,
-} from "./batch-get-item";
-import {
   BatchWriteItem,
-  createBatchWriteItemIntegration,
-  PutRequest,
-  DeleteRequest,
-} from "./batch-write-item";
-import {
-  createDeleteItemIntegration,
   DeleteItem,
-  DeleteItemReturnValues,
-  DeleteItemInput,
-  DeleteItemOutput,
-} from "./delete-item";
-import {
-  createGetItemIntegration,
   GetItem,
-  GetItemInput,
-  GetItemOutput,
-} from "./get-item";
-import {
-  createPutItemIntegration,
   PutItem,
-  PutItemInput,
-  PutItemOutput,
-} from "./put-item";
-import { createQueryIntegration, Query } from "./query";
-import { ReturnValues } from "./return-value";
-import { createScanIntegration, Scan } from "./scan";
-import type { Table } from "./table";
-import {
-  createTransactGetItemsIntegration,
+  Query,
+  ReturnValues,
   TransactGetItems,
-  TransactGetItem,
-  Get,
-} from "./transact-get-item";
-import {
-  createTransactWriteItemsIntegration,
   TransactWriteItems,
-  TransactWriteItemsInput,
-  TransactWriteItemsOutput,
-  ConditionCheck,
-  Update,
-  Put,
-  Delete,
-} from "./transact-write-item";
-import {
-  createUpdateItemIntegration,
   UpdateItem,
-  UpdateItemInput,
-  UpdateItemOutput,
-} from "./update-item";
-
-declare const a: AWS.DynamoDB;
+} from "@functionless/aws-dynamodb";
+import { aws_dynamodb } from "aws-cdk-lib";
+import { JsonFormat } from "typesafe-dynamodb";
```

Comment:
other places seem to import this from `lib`, why is this different?

Reply:
I can't remember, not something I intend to fix right now.